ccd-definition/CaseEventToComplexTypes/upload-c2/C2DocumentBundle-PREPARE_FOR_HEARING.json

```diff
@@ -7,7 +7,7 @@
     "ListElementCode": "document",
     "EventElementLabel": "Upload a file",
     "FieldDisplayOrder": 1,
-    "DisplayContext": "OPTIONAL"
+    "DisplayContext": "MANDATORY"
```

Comment:
Should we keep this? CCD will enable the fix in December. Probably no harm in leaving it.

Reply:
I think no harm in leaving this in